composer.json

```diff
@@ -4,9 +4,38 @@
     "license": "BSD-3-Clause",
     "keywords": [
         "zf",
+        "zendframework",
         "coding standard"
     ],
     "require": {
-        "squizlabs/php_codesniffer": "^2.7"
+        "squizlabs/php_codesniffer": "3.2.2"
```

Comment:
I think this is meant to be `^3.2.2`

Reply:
@Xerkus actually I put `3.2.2` on purpose, but we can change it.

My thought was to have very stable library, which will always behave the same, independently to dependencies. Sometimes change in dependency could contain a fix which will change behaviour of some sniffs and report error in places where before everything was fine.

Reply:
You have `~3.2.2` for that. It will lock to the 3.2.x branch, with a minimum of 3.2.2.

Reply:
@xtreamwayz I know, but still it does not guarantee that change in 3.2.x version is not affecting our sniffs/rules ;-)